src/main/java/org/gridsuite/study/server/StudyConstants.java

```diff
@@ -53,4 +55,21 @@ public enum SldDisplayMode {
         FEEDER_POSITION,
         STATE_VARIABLE;
     }
+
+    public enum EquipmentType {
+        GENERATOR,
+        LINE,
+        SUBSTATION,
+        TWT,
+        THWT,
+        BATTERY,
+        DANGLING_LINE,
+        HVDC_LINE,
+        LCC_CONVERTER_STATION,
+        LOAD,
+        SHUNT_COMPENSATOR,
+        STATIC_VAR_COMPENSATOR,
+        VSC_CONVERTER_STATION,
+        VOLTAGE_LEVEL;
+    }
```

Comment:
Why not using IdentifiableType here ?

Reply:
because we are missing 2 types `LCC_CONVERTER_STATION` and `VSC_CONVERTER_STATION`